src/lib.rs

```diff
@@ -143,7 +143,7 @@ impl FixedBitSet
     #[inline]
     pub fn insert(&mut self, bit: usize)
     {
-        assert!(bit < self.length);
+        assert!(bit < self.length, "unable to insert a bit at position {} in a fixedbitset of size {}", bit, self.length);
```

Comment:
This is somewhat of a nitpick, but I would prefer a more concise message such as `insert at index {} exceeds fixbitset size {}`.

Reply:
Changed in f2f2540 .

Also, edits on this PR are allowed to maintainers. Feel free to make any change you want.